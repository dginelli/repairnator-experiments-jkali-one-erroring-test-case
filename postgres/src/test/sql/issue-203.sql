select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*';
select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(_score desc, 0, 10) beer and title:*';

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 10, 10) java and title:*' order by 1 asc;
