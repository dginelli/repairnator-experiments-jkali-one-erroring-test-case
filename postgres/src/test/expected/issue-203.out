select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*';
 count 
-------
    10
(1 row)

select count(*) from so_posts where zdb('so_posts', ctid) ==> '#limit(_score desc, 0, 10) beer and title:*';
 count 
-------
    10
(1 row)

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 0, 10) java and title:*' order by 1 asc;
  id  
------
   16
  262
  368
  370
  663
  667
  812
 1189
 1257
 1674
(10 rows)

select id from so_posts where zdb('so_posts', ctid) ==> '#limit(id asc, 10, 10) java and title:*' order by 1 asc;
  id  
------
 1719
 1750
 3131
 3519
 3645
 4296
 4546
 4550
 4662
 4714
(10 rows)

